pkg/kubectl/cmd/top.go

```diff
@@ -32,13 +32,15 @@ var (
 	topLong = templates.LongDesc(`
 		Display Resource (CPU/Memory/Storage) usage.
 
-		The top command allows you to see the resource consumption for nodes or pods.`)
+		The top command allows you to see the resource consumption for nodes or pods.
+
+		This command is available only if the Heapster works fine on the server. `)
```

Comment:
"This command requires Heapster to be correctly configured and working on the server."